Backends/PDF/paper.lisp

```diff
@@ -35,13 +35,22 @@
       (rotatef width height))
     (make-rectangle* 0 0 width height)))
 
+(defparameter *pdf-top-margin* 1)
+(defparameter *pdf-left-margin* 1)
+(defparameter *pdf-bottom-margin* 1)
+(defparameter *pdf-right-margin* 1)
+
```

Comment:
do we intend to make the margin size customizable? if not, we could just put below `(- height 2)` etc with a brief comment.